uber/model_checks.py

```diff
@@ -25,6 +25,14 @@ def duplicate_admin(account):
                 return 'That attendee already has an admin account'
 
 
+@validation.AdminAccount
+def duplicate_admin(account):
```

Comment:
Good catch!  Yeah, this should definitely be changed to say something like ``def has_email_addr`` or something.